LINK/usr/bin/generate_rpm_manifest.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+manifest_dir=$APPLIANCE_SOURCE_DIRECTORY/../manifest
+mkdir -p $manifest_dir
+
+pushd $manifest_dir
+  manifest=rpm_manifest_$(date +"%m%d%y").csv
+
+  # dnf command gives yum repo used to install rpm
+  dnf list --installed | tr -s " " | while read name_arch version license
+  do
+    if [[ "$name_arch" == "Installed" ]]; then continue; fi
+    IFS=. read name arch <<< $name_arch
+    echo -e "$name\t$version\t$license" >> dnf_list
+  done
+  sort -o dnf_list dnf_list
```

Comment:
Isn't the output from `dnf` already sorted?

Reply:
`rpm` and `dnf` sort slightly differently and that causes `join` to not work properly unless dnf output is sorted separately.